src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -22,6 +29,27 @@ const ChartsHolder = (props: {
     // const rtData = projection.getDataset('rtRange');
   }
 
+  const endDate = new Date('2020-05-15');
+
+  const rtRangeData =
+    projection &&
+    projection.getDataset('rtRange').data.map(d => ({
+      x: d.x,
+      y: d.y?.rt,
+      low: d?.y.low,
+      hi: d.y?.high,
+    }));
+
+  const testPositiveData =
+    projection &&
+    projection
+      .getDataset('testPositiveRate')
+      .data.map(d => ({ x: d.x, y: d.y }));
```

Comment:
You probably don't need this map here, the data is not really being transformed

Reply:
Ooops, haha.  Yeah, that was me being sloppy when removing the date transformation I had before..  Thanks!